report-ng/app/src/components/classes/classes.html

```diff
@@ -21,7 +21,6 @@
 
 <template>
     <require from="../../value-converters/chip-name-value-converter"></require>
```

Comment:
Is this still necessary?

Reply:
Yes it is because the values in `filters` cannot be directly used. Depending on the chip type it needs to be for example "Custom Filter" or "Passed" and for status and Class we need to use other ValueConverter additionally to display the chip labels correctly. All this is handled in the ChipNameValueConverter